app/views/shared/_notification.html.erb

```diff
@@ -34,6 +34,9 @@
             <path d="M18 10C18 14.4183 14.4183 18 10 18C5.58172 18 2 14.4183 2 10C2 5.58172 5.58172 2 10 2C14.4183 2 18 5.58172 18 10ZM3.6 10C3.6 13.5346 6.46538 16.4 10 16.4C13.5346 16.4 16.4 13.5346 16.4 10C16.4 6.46538 13.5346 3.6 10 3.6C6.46538 3.6 3.6 6.46538 3.6 10Z" fill="#E5E5E5" />
             <circle class="origin-center -rotate-90 animate-[stroke-fill_5s_300ms_forwards]" stroke="#141414" stroke-opacity="0.4" r="7.2" cx="10" cy="10" stroke-dasharray="43.9822971503" stroke-dashoffset="43.9822971503" />
           </svg>
+          <div class="absolute -top-2 -right-2">
+            <%= lucide_icon "x", class: "w-5 h-5 p-0.5 hidden group-hover:inline-block border border-alpha-black-50 border-solid rounded-[10px] bg-white text-gray-400 cursor-pointer", data: { action: "click->element-removal#remove" } %>
```

Comment:
To get the `x` to look correctly I had to create a border with round corners.  I tried the `circle-x` icon, but I couldn't change the circle colour to match the [supplied video](https://github.com/maybe-finance/maybe/discussions/874#discussioncomment-9817770).

To get the close button to appear when you move the mouse over the notification, like the video, I hid it first then I showed it with `group-hover:inline-block`.